server.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import "github.com/gin-gonic/gin"
+
+var get = make(map[string]string) // Map with user value
+
+func main() {
+	s := gin.Default() // Creates a Gin server
+	s.GET("/hello", func(c *gin.Context) { // Returns JSON
+		c.JSON(200, gin.H{
+			"message": "Hello world!", // Display simple "Hello world!"
+		})
+	})
+	s.GET("/greet/:name", func(c *gin.Context) { // Returns JSON with an argument from the user
```

Comment:
```suggestion
	s.GET("/greet/:"+NameParam, func(c *gin.Context) { // Returns JSON with an argument from the user
```
Make `"name"` a global const, declared above this function.